README.md

```diff
@@ -10,6 +10,34 @@ The majority of documentation on the site is drawn from [the probot docs](https:
 
 We also use this website to show off some of the cool probot plugins built by the community. If you would like to have your plugin listed there, check out our [Contributing Guidelines](https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md) for more information about what the requirements are to do that.
 
+## Install and Usage
+
+If you'd like to work on this repository, clone it. We use [Jekyll](https://jekyllrb.com/) to build the website, hosted on [GitHub Pages](https://pages.github.com/). We also use Git submodules to pull docs from [probot/docs](http://github.com/probot/docs).
```

Comment:
This should probably link to https://github.com/probot/probot/tree/master/docs and maybe instead of 'probot/docs' have it be 'probot/probot docs folder' since we don't have a docs repo, but open to other suggestions here.